gene/schemas.py

```diff
@@ -507,3 +636,131 @@ def schema_extra(schema: Dict[str, Any],
                     'url': 'https://github.com/cancervariants/gene-normalization'  # noqa: E501
                 }
             }
+
+
+class NormalizeService(BaseModel):
+    """Define model for returning normalized concept."""
+
+    query: str
+    warnings: Optional[List]
```

Comment:
does the List always contain Dicts or can it contain strings too?

Reply:
(I think this would apply to the other service model as well?)

Reply:
Good question. I think right now it's just Dicts